src/components/roll-engine/engine.js

```diff
@@ -469,11 +469,11 @@ export class FBLRollHandler extends FormApplication {
 		// however Infinity is finicky to serialize.
 		// So we use a sufficiently large number instead.
 		// eslint-disable-next-line no-nested-ternary
-		const maxPush = unlimitedPush
-			? 10000
-			: game.actors.get(this.options.actorId).system.type === "monster"
-				? "0"
-				: 1;
+		const currentActor = game.actors.get(this.options.actorId);
+		const isMonster = currentActor
```

Comment:
```suggestion
		const isMonster = currentActor?.system.type === "monster";
```
Using the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) will make this code more readable.

Reply:
Thanks a lot. JS is very much not my cup of tea, so all suggestions are greatly appreciated.